cleanup/Jenkinsfile

```diff
@@ -0,0 +1,63 @@
+import groovy.json.JsonSlurper
+
+pipeline {
+    agent none
+    options {
+        timeout(time: 1, unit: 'HOURS')
+        disableConcurrentBuilds()
+    }
+    stages {
+        stage('Cleanup') {
+            parallel {
+                stage('Cleanup Windows temp directory') {
+                    agent {
+                        label 'windows'
+                    }
+                    steps {
+                        script {
+                            cleanDirs('tmp/yarn', '\"%temp%\"', '(yarn--*)')
```

Comment:
I wonder if it would be safe to remove the whole of the content from `%temp%`? Maybe not.

Reply:
I am not sure. I would assume that other tools running on the system use the temp dir as well, so we might run into issues with permissions or cause unwanted side-effects.